src/aat/resources/features/F-049/S-511.td.json

```diff
@@ -0,0 +1,23 @@
+{
+  "_guid_": "S-511",
+  "_extends_": "F-049_Test_Data_Base",
+  "title": "must return 403 when request provide invalid Jurisdiction ID in CCD",
+  "specs": [
+    "a detailed profile in CCD",
+    "Start case creation as Case worker",
+    "CCD Data Store",
```

Comment:
These are places in their own attributes in test data object

Reply:
as "Start case creation as Case worker", and "CCD Data Store" can be a part of base.td.json but due to specs is an array we can put into base.td.json or if you can suggest some other option will change

Reply:
anyway.. fixed myself.